app/backend/setup-pg.js

```diff
@@ -0,0 +1,35 @@
+const pg = require('pg');
+const session = require('express-session');
+const PgSession = require('connect-pg-simple')(session);
+
+const PGUSER = process.env.PGUSER || 'rootadmin';
+
+let databaseURL = 'postgres://';
+
+databaseURL += PGUSER;
+if (process.env.PGPASSWORD) {
+  databaseURL += `:${process.env.PGPASSWORD}`;
+}
+
+databaseURL += '@';
+
+databaseURL += process.env.PGHOST || 'localhost';
+if (process.env.PGPORT) {
+  databaseURL += `:${process.env.PGPORT}`;
+}
+
+databaseURL += '/';
+databaseURL += process.env.PGDATABASE || 'connectivity-intake';
+
+const connectPgPool = () => {
+  const pgPool = new pg.Pool({ connectionString: databaseURL });
+
+  const store = new PgSession({
+    pool: pgPool,
+    // tableName: 'connect_session',
```

Comment:
without passing in a tableName this will default to a table named `session` I think. Still has to be created in advance. I'd lean towards being verbose and passing in the table name.

Reply:
It's really important for us to pass in a cookie secret from our environment

Reply:
We'll also need to make sure the cookie max age is longer than whatever our session is set as

Reply:
I think we're using `process.env.SESSION_SECRET` for our cookie secret elsewhere